<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)</title>
  <meta name="description" content="Chapter 1 Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  <meta name="generator" content="bookdown 0.26.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Methodology | Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)" />
  
  
  

<meta name="author" content="Shruti Sridhar" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="introduction.html"/>
<link rel="next" href="2-results-and-discussion.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#objective"><i class="fa fa-check"></i>Objective</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html"><i class="fa fa-check"></i><b>1</b> Methodology</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#datasets"><i class="fa fa-check"></i><b>1.1</b> Datasets</a></li>
<li class="chapter" data-level="1.2" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#immunedeconvolution"><i class="fa fa-check"></i><b>1.2</b> Immunedeconvolution</a></li>
<li class="chapter" data-level="1.3" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#pathway-analysis"><i class="fa fa-check"></i><b>1.3</b> Pathway Analysis</a></li>
<li class="chapter" data-level="1.4" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#sting-correlation-analysis"><i class="fa fa-check"></i><b>1.4</b> STING correlation analysis</a></li>
<li class="chapter" data-level="1.5" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#statistical-analysis"><i class="fa fa-check"></i><b>1.5</b> Statistical analysis</a></li>
<li class="chapter" data-level="1.6" data-path="1-rmd-basics.html"><a href="1-rmd-basics.html#clustering-analysis"><i class="fa fa-check"></i><b>1.6</b> Clustering analysis</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-results-and-discussion.html"><a href="2-results-and-discussion.html"><i class="fa fa-check"></i><b>2</b> RESULTS AND DISCUSSION:</a></li>
<li class="chapter" data-level="3" data-path="3-ref-labels.html"><a href="3-ref-labels.html"><i class="fa fa-check"></i><b>3</b> Graphics, References, and Labels</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-ref-labels.html"><a href="3-ref-labels.html#figures"><i class="fa fa-check"></i><b>3.1</b> Figures</a></li>
<li class="chapter" data-level="3.2" data-path="3-ref-labels.html"><a href="3-ref-labels.html#footnotes-and-endnotes"><i class="fa fa-check"></i><b>3.2</b> Footnotes and Endnotes</a></li>
<li class="chapter" data-level="3.3" data-path="3-ref-labels.html"><a href="3-ref-labels.html#bibliographies"><i class="fa fa-check"></i><b>3.3</b> Bibliographies</a></li>
<li class="chapter" data-level="3.4" data-path="3-ref-labels.html"><a href="3-ref-labels.html#anything-else"><i class="fa fa-check"></i><b>3.4</b> Anything else?</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="B" data-path="B-the-second-appendix-for-fun.html"><a href="B-the-second-appendix-for-fun.html"><i class="fa fa-check"></i><b>B</b> The Second Appendix, for Fun</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Identifying immune features that determine immunogenicity and influence prognosis in chemotherapy treated diffuse large B cell lymphoma (DLBCL)</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="rmd-basics" class="section level1 hasAnchor" number="1">
<h1><span class="header-section-number">Chapter 1</span> Methodology<a href="1-rmd-basics.html#rmd-basics" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="datasets" class="section level2 hasAnchor" number="1.1">
<h2><span class="header-section-number">1.1</span> Datasets<a href="1-rmd-basics.html#datasets" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The following whole genome RNA-Seq DLBCL datasets were compiled for analysis and validation -</p>
<ul>
<li><p><em>GOYA DLBCL dataset</em></p>
<ul>
<li>This dataset contains collected RNASeq data of 552 samples from the GOYA clinical trial. The GOYA trial tested the efficacy of Gazyva (GA101) compared with Rituxan (Rituximab) in first line, untreated DLBCL patients <span class="citation">(Nowicka et al., 2021)</span>.</li>
</ul></li>
<li><p><em>Dave dataset</em></p>
<ul>
<li>This dataset contain exome sequencing data for 1001 DLBCL patients and RNA sequencing data for 775 DLBCL patients <span class="citation">(Reddy et al., 2017)</span>.</li>
</ul></li>
<li><p><em>Staudt dataset</em></p>
<ul>
<li>This retrospective study included 181 clinical samples from CHOP-treated patients and 233 clinical samples from Rituximab-CHOP-treated patients <span class="citation">(Schmitz et al., 2018)</span>.</li>
</ul></li>
</ul>
<p>The following microarray datasets were choosen for analysis -</p>
</div>
<div id="immunedeconvolution" class="section level2 hasAnchor" number="1.2">
<h2><span class="header-section-number">1.2</span> Immunedeconvolution<a href="1-rmd-basics.html#immunedeconvolution" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Immune deconvolution by the following methods was conducted for the chosen datasets.</p>
<ul>
<li>quanTIseq</li>
<li>CIBERSORT</li>
<li>MCP - Counter</li>
<li>xCell</li>
<li>EPIC</li>
</ul>
</div>
<div id="pathway-analysis" class="section level2 hasAnchor" number="1.3">
<h2><span class="header-section-number">1.3</span> Pathway Analysis<a href="1-rmd-basics.html#pathway-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>CIBERSORT results were chosen for further downstream analysis due to better resolution of the immune subtypes. For each immune subtype, the patient samples were divided into two cohorts (high and low) based on the immune cell subtype infiltration score (CIBERSORT Score). DeSeq223 analysis was conducted to find the differentially expressed genes following which Gene Set Enrichment24 Analysis using Hallmark gene sets was performed. R 4.1.1 and GraphPad PRISM 9 were used for visualization.</p>
</div>
<div id="sting-correlation-analysis" class="section level2 hasAnchor" number="1.4">
<h2><span class="header-section-number">1.4</span> STING correlation analysis<a href="1-rmd-basics.html#sting-correlation-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The CIBERSORT Score for each immune cell type was divided into quartiles (CIBERSORT high (Q4), Interquartile range (IQR) and CIBERSORT low (Q1)) and correlated with STING TPM counts for each dataset. Violin plots were used to visualize the results. STING expression for each dataset was divided into quartiles and GSEA analysis was performed between Quartile 1 and Quartile 4 in each of the microarray datasets in order to evaluate pathway level differences.</p>
</div>
<div id="statistical-analysis" class="section level2 hasAnchor" number="1.5">
<h2><span class="header-section-number">1.5</span> Statistical analysis<a href="1-rmd-basics.html#statistical-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Descriptive statistics and one way ANOVA tests were conducted using GraphPad PRISM 9 software for evaluating statistical significance of each quartile. FDR &lt; 0.05 and p value &lt; 0.05 were considered statistically significant for GSEA and GO analysis. Pearson correlation was conducted using R 4.1.1.</p>
</div>
<div id="clustering-analysis" class="section level2 hasAnchor" number="1.6">
<h2><span class="header-section-number">1.6</span> Clustering analysis<a href="1-rmd-basics.html#clustering-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>25 FGES (Functional Gene Enrichment Signatures) signatures38 were used to identify microenvironmental patterns among DLBCL samples, stratified by STING expression (quartile 4 being STING High samples and quartile 1 Low STING samples) by unsupervised dense clustering using the Louvain method for community detection. Datasets GSE117556, GSE32918, GSE10846, GSE23501 and GSE31312 were individually analyzed and Gene Set Variation Analysis (GSVA) was conducted to generate activity scores.</p>
<p>Intersample similarity of the activity scores was calculated using Pearson correlation. The resulting distance matrix was converted into a graph where each sample formed a node and two nodes formed an edge with weight equal to the pair’s Pearson correlation.</p>
<p>To mathematically determine the optimum threshold for observed clusters for each dataset, maximum Calinski Harabasz and Silhouette scores were used excluding separations with low-populated clusters. The Calinski-Harabasz index, also known as the Variance Ratio Criterion, is the ratio of the sum of between-clusters dispersion and of inter-cluster dispersion for all clusters, the higher the score, the better the performance.</p>
<p>This score is fast to compute and the score is higher when clusters are dense and well separated. Silhouette Coefficient or silhouette score is a metric used to calculate the goodness of a clustering technique. Its value ranges from -1 to 1. WIth 1 meaning the clusters are well apart from each other and clearly distinguished. 0 meaning the clusters are indifferent and -1 meaning the clusters are assigned in the wrong way. The Silhouette score was used as validation after computing the Calinski-Harabasz index and optimum threshold was decided.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-results-and-discussion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
